docs/semgrep-app/rule-board.md

```diff
@@ -1,113 +1,89 @@
 ---
 slug: rule-board
 append_help_link: true
+description: "The Rule Board is a visual representation of the rules that Semgrep App uses to scan code. Rules are cards, and are grouped into columns representing the actions undertaken (whether to block, comment, or silently monitor) when a finding surfaces."
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
 # Rule board
 
-The rule board is a visual representation of the rules
-that Semgrep App uses to scan code in CI.
 
-Rules and rulesets are displayed as cards,
-which you can drag and drop between columns.
-The placement of cards in columns determines the Semgrep rules that run in CI and the actions taken in response to findings.
+The Rule Board is a visual representation of the rules that Semgrep App uses to scan code. Rules can be organized into rulesets. Rulesets are rules related through programming language, OWASP category, framework, and so on.
```

Comment:
```suggestion
The Rule Board is a visual representation of the rules that Semgrep App uses to scan code. Rules can be organized in rulesets. Rulesets are rules related through a programming language, OWASP category, or framework.
```

What more can we say instead of "and so on" at the end? We can avoid it as well. Search for "etc." in https://developers.google.com/style/word-list?hl=en